docker/docker-compose-mini.yml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2016, EMC, Inc.
+
+version: '2'
+
+volumes:
+  dhcp-leases:
+    external: false
+
+services:
+
+  mongo: # 27017
+    image: mongo:latest
+    network_mode: "host"
+    privileged: true
```

Comment:
Why is this running privileged and network host mode?  Can you run it unprivileged and use the DNS capabilities of docker and the dependency chaining of docker-compose to run this as an isolated container instead.  Same comment for rabbitmq below.

ie: 
  mongo: # 27017
    image: mongo:latest
    expose:
      - "27017"
  http: # 9090, 9080
    build: "../on-http"
    links:
      - mongo
      - rabbitmq
    ports:
      - "9090:9090"
      - "9080:9080"
    image: rackhd/on-http:latest
    volumes:
      - "./files/mount/common:/RackHD/on-http/static/http/common"
      - "./monorail:/opt/monorail"

and change the monorail/config.json entry to:
    "mongo": "mongodb://mongo/rackhd",